src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import cn from 'classnames';
+
+// eslint-disable-next-line max-len
+const checkIfTabExists = (list, Id) => list.filter(tab => tab.id === Id).length > 0;
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTabID = checkIfTabExists(tabs, selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
```

Comment:
make your ternaries as simple as it possible, write the result of the function into variable, for example `const isTabExists = checkIfTabExists(tabs, selectedTabId)`